pkg/storage/fs/lister_test.go

```diff
@@ -1,10 +1,20 @@
 package fs
 
+import (
+	"context"
+
+	"github.com/gobuffalo/buffalo"
+)
+
 func (d *FsTests) TestList() {
+	c := &buffalo.DefaultContext{
```

Comment:
nit: we should create a new context on each test run. can use [`SetupTestSuite`](https://godoc.org/github.com/stretchr/testify/suite#SetupTestSuite) for that

Reply:
Fixed locally.

I didn't use SetupTestSuite, but I did add a span to each of the tests that executes against functions that I've touched so far.

Reply:
Ahh... looks like I do need to SetupTestSuite to initialize opentracing. As per buffet's README. I'm getting there don't worry about me.